tools/run_tests/sanity/check_buildifier.sh

```diff
@@ -0,0 +1,20 @@
+#! /usr/bin/env bash
+# Copyright 2019 The gRPC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
```

Comment:
Is the `-x` necessary or desirable here?

Reply:
I added the debug flag helps us to trace problem. It's better than noop IMO. What do you think?